data/hai/hai reveal 4 middle.txt

```diff
@@ -1638,7 +1638,7 @@ mission "Hai Reveal [A11-C] Outgoing Election Update 1"
 		not planet "Hai-home"
 		attributes "spaceport"
 	to offer
-		has "Hai Reveal [A11] Sound the Alarm: offered"
+		has "Hai Reveal [A11] Sound the Alarm: active"
```

Comment:
The election doesn't actually seem to conclude until the next mission is done/the one after it is offered. So, maybe:
```suggestion
		has "Hai Reveal [A11] Sound the Alarm: offered"
		not "Hai Reveal [A12] Into the Lion's Den: done"
```
Or:
```suggestion
		has "Hai Reveal [A11] Sound the Alarm: offered"
		not "Hai Reveal [A13] A New Elder: offered"
```
Or, have completion of the next mission `fail` this one?

Reply:
It's a bit vague, but I think it can be argued that the election is still going until A13. That means your second option is correct. I will apply the change manually and test it.

Reply:
It works.